### PR TITLE
Soundeffects

### PR DESCRIPTION
**Pull request by nevyn**

### CODE REVIEW

**Comment by jonasringblom** [comment]
nice.

Den ons 31 mars 2021 kl 23:23 skrev Nevyn Bengtsson <
***@***.***>:

File: lua/eng/assets_eng.lua [hunk 1]
```diff
@@ -6,12 +6,72 @@ namespace("networkscene", "alloverse")
...
+function AssetsEng:loadFromAsset(asset, type, callback)
+    if asset._lovrObject then 
+      callback(asset._lovrObject)
+      return
+    end
+    if asset._lovrObjectLoadingCallbacks then
+      table.insert(asset._lovrObjectLoadingCallbacks, callback)
+      return
+    end
+    asset._lovrObjectLoadingCallbacks = {callback}
```
Comment (Voxar) [inline lua/eng/assets_eng.lua:53]: This should probably be in an else

Comment (Voxar) [reply lua/eng/assets_eng.lua:53]: Ops nvm saw the return now. Why do I code this way :D

Comment (nevyn) [reply lua/eng/assets_eng.lua:53]: ;D that's how I code, you copied my atrocious style perhaps? ;)

**Pull request merged**